src/main/java/lz/modelTesting/configuration/MongoDBConfig.java

```diff
@@ -12,13 +12,14 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.data.mongodb.repository.config.EnableMongoRepositories;
 
-import java.util.Optional;
+import java.util.*;
```

Comment:
Not a problem but be aware of it:
https://stackoverflow.com/questions/147454/why-using-a-wild-card-with-a-java-import-statement-bad